integration/src/client.spec.ts

```diff
@@ -112,4 +116,23 @@ baz`,
       expect(result.matches?.length).toEqual(11);
     });
   });
+
+  describe('apl tabular query', () => {
+    it('returns a valid response', async () => {
+      const status = await axiom.ingest(datasetName, { test: 'apl' });
+      expect(status.ingested).toEqual(1);
+      // wait 1 sec for ingestion to finish
+      await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
nit: would be nice to use fake timers so the tests dont have to actually wait for this.

Reply:
Hmmm, I tested it without sleep and with fake timers and couldn't get it to work (skill issue?). It seems like this sleep ensures that it doesn't test against data that hasn't been written in the dataset yet. It is also using the without batching client, so there aren't any timeouts in the client